src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -166,20 +164,48 @@ static void SetControlPropertiesFromProxy(UIButton platformView)
 			}
 		}
 
-		void OnButtonTouchUpInside(object? sender, EventArgs e)
+		class ButtonEventProxy
```

Comment:
There are several controls that can benefit from a similar change. I can create a list with the ones with the greatest impact.